### ISSUE TITLE
Accessibility declaration

### ISSUE THREAD

**Issue by dascritch** [cpuprogramme/cpu-15#68]
Recommended by @astranchet and she is right
https://twitter.com/annso_/status/1417739759577968643
(even if we cannot be compliant due to non-transcived interviewes and missing chronicles)

**Comment by astranchet** [comment]
Yes, a statement is useful even if (especially if) some part of the product are not accessible. It allows to:
- reference innaccessible functionnality/content (and possible workaround)
- make a statement about coming up features, etc.
- give an opportunity for users to keep in touch about the subjects

**Comment by dascritch** [comment]
We have : 
- [an accessibility declaration](https://cpu.dascritch.net/pages/Accessibilit%C3%A9-%3A-partielle)
- [a tag indicating mission transcription](https://cpu.dascritch.net/tag/transcription%20manquante)
